cloud_governance/common/helpers/aws/aws_cleanup_operations.py

```diff
@@ -0,0 +1,153 @@
+import datetime
+
+import boto3
+
+from cloud_governance.common.clouds.aws.s3.s3_operations import S3Operations
+from cloud_governance.common.helpers.cleanup_operations import AbstractCleanUpOperations
+from cloud_governance.common.logger.init_logger import logger
+
+
+class AWSCleanUpOperations(AbstractCleanUpOperations):
+
+    def __init__(self):
+        super().__init__()
+        self._region = self._environment_variables_dict.get('AWS_DEFAULT_REGION', 'us-east-2')
+        self.__s3operations = S3Operations(region_name=self._region)
+        self._ec2_client = boto3.client('ec2', region_name=self._region)
+        self._s3_client = boto3.client('s3')
+        self._iam_client = boto3.client('iam')
+
+    def get_tag_name_from_tags(self, tags: list, tag_name: str) -> str:
+        if tags:
+            for tag in tags:
+                if tag.get('Key').strip().lower() == tag_name.lower():
+                    return tag.get('Value').strip()
+        return ''
+
+    def get_clean_up_days_count(self, tags: list):
```

Comment:
Seems it also updates the cleanup days value ?

Reply:
No, It returns the clean_up days count. We have another method which updates the days count.
[Check here](https://docs.google.com/document/d/1hPv80Xe6-Cao59smi3Yougsa9lfhYparpxVTmNpIM94/edit).